pkg/autostart/io.lima-vm.autostart.INSTANCE.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>io.lima-vm.autostart.{{ .Instance }}</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>{{ .Binary }}</string>
+		<string>start</string>
+		<string>--foreground</string>
+		<string>--pidfile</string>
+		<string>ha.pid</string>
+		<string>--socket</string>
+		<string>ha.sock</string>
+		<string>{{ .Instance }}</string>
+	</array>
+	<key>RunAtLoad</key>
+	<true/>
+	<key>StandardErrorPath</key>
+	<string>ha.stderr.log</string>
+	<key>StandardOutPath</key>
+	<string>ha.stdout.log</string>
```

Comment:
The outputs of `limactl start` are lost when `limactl start` and `limactl hostagent` output to the same files, so they need to be separated.
```suggestion
	<key>StandardErrorPath</key>
	<string>launchd.stderr.log</string>
	<key>StandardOutPath</key>
	<string>launchd.stdout.log</string>
```

Reply:
Thank you for the tip 🙏